istio-model/pom.xml

```diff
@@ -46,6 +46,12 @@
       <artifactId>joda-time</artifactId>
       <version>2.9.9</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.joda/joda-convert -->
+    <dependency>
+      <groupId>org.joda</groupId>
+      <artifactId>joda-convert</artifactId>
```

Comment:
Is this really needed? I've been wondering about it, actually, so curious about it.

Reply:
Yes it is needed. It doesn't compile without it. It complains a class called FromString is missing in Joda.

Reply:
May I ask how you are compiling since I've never seen that issue? The only time something complained about that class is my IDE. Building from the command line never ran into this.

Reply:
Actually I always compile using command line. It started complaining once then continued complaining, but that class is in joda-convert so it is normal to complain.